wheel/python/chia_rs/chia_rs.pyi

```diff
@@ -3745,10 +3745,10 @@ class RespondPuzzleState:
         coin_states: Union[ List[CoinState], _Unspec] = _Unspec()) -> RespondPuzzleState: ...
 
 class RejectPuzzleState:
-    reason: uint8
+    reason: int
```

Comment:
this doesn't look right now. I suppose it doesn't matter much as long as we rely on serializing and de-serializing this on the rust side